python/cudf/cudf/tests/test_orc.py

```diff
@@ -388,7 +389,7 @@ def test_orc_writer(datadir, tmpdir, reference_file, columns, compression):
     cudf.from_pandas(expect).to_orc(gdf_fname.strpath, compression=compression)
     got = pa.orc.ORCFile(gdf_fname).read(columns=columns).to_pandas()
 
-    assert_eq(expect, got)
+    assert_frame_equal(cudf.from_pandas(expect), cudf.from_pandas(got))
```

Comment:
If these are large dataframes, we might want to avoid calling `cudf.from_pandas(expect)` more times than necessary in this and related tests. (This snippet will need to be reformatted, sorry.)

```python
    expect = cudf.from_pandas(orcfile.read(columns=columns).to_pandas())
    expect.to_orc(gdf_fname.strpath, compression=compression)
    got = cudf.from_pandas(pa.orc.ORCFile(gdf_fname).read(columns=columns).to_pandas())

    assert_frame_equal(expect, got)
```